__fixtures__/javascript.js

```diff
@@ -42,4 +44,4 @@ export function shadow() {
 
 nonExistent();
 
-// There is intentionally no new line at the end of this file to hit [eol-last](https://eslint.org/docs/rules/eol-last).
\ No newline at end of file
+// There is intentionally no new line at the end of this file to hit [eol-last](https://eslint.org/docs/rules/eol-last).
```

Comment:
revert

Reply:
😓  Yeah I need to make a commit just for this one - on-commit actions provided by my editor was doing stuff.